### ISSUE TITLE
System.Drawing.Common 4.5r on Xamarin.Android -> System.PlatformNotSupportedException: System.Drawing is not supported on this platform.

### ISSUE THREAD

**Issue by TerceraFaseSoftware** [dotnet/corefx#29761]
Hi!

I'm working on a Xamarin.Android application and when I try to use System.Drawing.Bitmap, I get this exception at run time. 

         System.PlatformNotSupportedException: System.Drawing is not supported on this platform.

My question is:
Is this library compatible with Xamarin.Android? And, if it is not, when will it be?

Because, if it is not compatible, why can I compile and execute it without any problem until the moment I generate a new Bitmap?

Thanks.

**Comment by safern** [comment]
This is related to: https://github.com/mono/mono/issues/8089#issuecomment-387461129

I think for now this is the intended behavior in mobile as it is not yet supported. 

cc: @marek-safar @akoeplinger

**Comment by marek-safar** [comment]
Yes, this is one of those types/methods which are unlikely to be fully supported cross-platform.

**Comment by BorzillaR** [comment]
I have the same question but for UWP apps. I create a blank UWP app, install the nuget package System.Drawing.Common v4.5.0-rc1 and create a new bitmap in the MainPage constructor like this: 

`var bm = new System.Drawing.Bitmap(100, 100, PixelFormat.Format32bppArgb)`

App is successfully compiled but failed at run time with PlatformNotSupportedException. Can it be used in UWP apps?

**Comment by danmosemsft** [comment]
@borzillar unfortunately no, that code predates UWP so much of it uses GDI which is not available in UWP. @pjanotti does your analysis tool flag this?

**Comment by BorzillaR** [comment]
@danmosemsft , thank you!

**Comment by danmosemsft** [comment]
@BorzillaR this is the tool I mentioned -- https://github.com/dotnet/platform-compat -- it's a VS extension, please try it out, it seems that it should flag this.

As for GDI+ on UWP - I am not aware of plans but I am not part of that team. I would find a different library to do what you need.

**Comment by smchughinfo** [comment]
![files](https://i.imgur.com/rmeVa74.png)

None of the iOS or Android directories have an implementation. Why are these even listed as an option in a Xamarin project?
